src/hotspot/share/runtime/threads.cpp

```diff
@@ -768,6 +769,11 @@ jint Threads::create_vm(JavaVMInitArgs* args, bool* canTryAgain) {
     NativeHeapTrimmer::initialize();
   }
 
+  // Start RSS watcher task if needed
+  if (RssLimit != nullptr) {
+    RssWatcher::initialize(RssLimit);
+  }
```

Comment:
So I think if we are on AIX or regular BSD then we should at least give a warning that the flag will be ignored, and actually ignore it.